index.bs

```diff
@@ -3263,7 +3261,8 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
     :: |source|'s [=trigger specs=]
 
 1. Let |epsilon| be |source|'s [=attribution source/event-level epsilon=].
-1. Let |states| be the [=obtain a set of possible trigger states|number of possible trigger states=] obtained with |randomizedResponseConfig|.
+1. Let |possibleValues| be the result of [=obtaining a set of possible trigger states=] with |randomizedResponseConfig|.
```

Comment:
Probably rename to possibleTriggerStates and numPossibleTriggerStates for clarity?